.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: ci
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 8, 10, 12 ]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm ci
+      - run: npm run compile
+      - run: npm run test
+  coverage:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm run compile
+      - uses: paambaati/codeclimate-action@v2.3.0
```

Comment:
why dropping `codecov` for `codeclimate`?

Reply:
Codecov PR comments are a bit intrusive imho (but then maybe they can be disabled?)

Code Climate has additional code quality checks (duplication, complexity, etc) on top of two different coverage checks (a coverage threshold (%) for changed code and an overall coverage diff).

Reply:
I'm okay with reverting back to Codecov, if we can get rid of the comments 😉